sbin/prepareWorkspace.sh

```diff
@@ -581,44 +581,69 @@ setupGpg() {
     echo "GNUPGHOME=$GNUPGHOME"
 }
 
-# Download the required DevKit if necessary
+# Download the required DevKit if necessary and not available in /usr/local/devkit
 downloadDevkit() {
   if [[ -n "${BUILD_CONFIG[USE_ADOPTIUM_DEVKIT]}" ]]; then
     rm -rf "${BUILD_CONFIG[WORKSPACE_DIR]}/${BUILD_CONFIG[WORKING_DIR]}/devkit"
     mkdir -p "${BUILD_CONFIG[WORKSPACE_DIR]}/${BUILD_CONFIG[WORKING_DIR]}/devkit"
 
-    local devkit_tar="${BUILD_CONFIG[WORKSPACE_DIR]}/${BUILD_CONFIG[WORKING_DIR]}/devkit/devkit.tar.xz"
+    BUILD_CONFIG[ADOPTIUM_DEVKIT_LOCATION]=""
 
-    setupGpg
-
-    # Determine DevKit tarball to download for this arch and release
-    local devkitUrl="https://github.com/adoptium/devkit-binaries/releases/download/${BUILD_CONFIG[USE_ADOPTIUM_DEVKIT]}"
     local devkit_target="${BUILD_CONFIG[OS_ARCHITECTURE]}-linux-gnu"
-    local devkit="devkit-${BUILD_CONFIG[USE_ADOPTIUM_DEVKIT]}-${devkit_target}"
 
-    # Download tarball and GPG sig
-    echo "Downloading DevKit : ${devkitUrl}/${devkit}.tar.xz"
-    curl -L --fail --silent --show-error -o "${devkit_tar}" "${devkitUrl}/${devkit}.tar.xz"
-    curl -L --fail --silent --show-error -o "${devkit_tar}.sig" "${devkitUrl}/${devkit}.tar.xz.sig"
+    local USR_LOCAL_DEVKIT="/usr/local/devkit"
+    if [[ -d "${USR_LOCAL_DEVKIT}" ]]; then
+      local usrLocalDevkitInfo="${USR_LOCAL_DEVKIT}/devkit.info"
```

Comment:
I'm currently set up to install into a subdirectory of /usr/local/devkit with the name of the particular devkit in order to allow different ones to be installed, so this will need to be customizable, or otherwise defined somewhere. I don't think we should be expecting a single devkit on the machien in that location, which looks like what this PR is expecting.
At the moment I have /usr/local/devkit/s390x-on-s390x for s390x and on the others it's a directory under /usr/local/devkit [matching the name of the release](https://github.com/adoptium/infrastructure/pull/3488/files#diff-fc137c7ee8ffb4d34a53f0f2484bcb64340a2b18fd885a1384bac9d06419eab8)

Reply:
ah ok, i'll add an extra build arg for "local devkit directory"

Reply:
I don't really want to have to add hard coded directory paths to the pipeline config, how about I search /usr/local/devkit/<devkit version tag>/

Reply:
I have updated local USR_LOCAL_DEVKIT="/usr/local/devkit" to include the DevKit release tag